cmd/orb-cli/common/common.go

```diff
@@ -520,10 +520,18 @@ func (s *Signer) Headers() jws.Headers {
 		headers[jws.HeaderAlgorithm] = "EdDSA"
 	}
 
-	if s.publicKey.Crv == "P-256" || s.publicKey.Crv == "P-384" || s.publicKey.Crv == "P-521" {
+	if s.publicKey.Crv == "P-256" {
 		headers[jws.HeaderAlgorithm] = "ES256"
 	}
 
+	if s.publicKey.Crv == "P-384" {
```

Comment:
It would be nicer to have a map of crv to algorithm.

Reply:
done